assets/scripts/cat-api.js

```diff
@@ -0,0 +1,13 @@
+this.baseUrl = 'https://api.thecatapi.com/v1/';
```

Comment:
Environment specific strings like this shouldn't be hard coded, and instead made an environment variable that will be added at build. `dotenv` is the usual library for this with JS.

Reply:
Furthermore, which `this` do you intend to mutate? It's likely not doing what you expect it to do, especially as this is an ES module.

At the least it should be `const baseUrl = '...'`.

Reply:
> Environment specific strings like this shouldn't be hard coded, and instead made an environment variable that will be added at build. `dotenv` is the usual library for this with JS.

Hi @lloydjatkinson , thanks for commenting on this PR. When we started this project we aimed to keep it simple and not use external libraries or such, hence we arent using dotenv here. And whilst this is technically correct- we are calling a public api that anyone anywhere can get access to. If we were using api keys , or it was a private/internal api then 100% we would be doing this differently and storing these in .env file. But it doesnt make sense to use it in this situation.

Reply:
> Furthermore, which `this` do you intend to mutate? It's likely not doing what you expect it to do, especially as this is an ES module.
> 
> At the least it should be `const baseUrl = '...'`.

Hi @lloydjatkinson , thanks for your comments, you are right to raise it as a query, but we like to give out appropriate feedback for the skill level of our members and explain what the change is and importantly why we are asking for that change.

We are all here to gain experience and try things out, and whilst the work might not be perfect we want members to have a positive experience and to feel good about the work they have done. Ruman has done a good job with this and has already taken the time to implement a change i have requested so i am happy to accept this PR as is, it is working and bringing back the results from the api call as per the ticket. 

@Loop-infinity - you may or may not be aware that the "this" keyword in Javascript can have different values depending on where it is being used and thus can produce unexpected results if not used correctly and this is what Lloyd is referring to.  If you wish to go over this then  i am more than happy to do so, but for now we will accept your PR and we can refactor this if it needs it.

Reply:
@lloydjatkinson Thanks for your feedback. I had first written everything in an object oriented way in a class using modules. But then I refactored it to keep it simple and probably forgot to remove the **this** keyword.

Reply:
@Loop-infinity Happy to help, I'm glad you took it on board. Things happen 😄